### PR TITLE
[IOTDB-658] Update JMX User Guide document

### PR DESCRIPTION
**Pull request by samperson1997**

### CODE REVIEW

**Comment by HTHou** [comment]
Would you like to use the english version screen shots in english doc? I can make them in my english operating system? :)

**Comment by samperson1997** [comment]
> Would you like to use the english version screen shots in english doc? I can make them in my english operating system. :)

Oh right! I ignored the language in the image! I'll change the pics.😂

File: docs/zh/UserGuide/6-System Tools/3-JMX Tool.md [hunk 1]
```diff
@@ -23,32 +23,33 @@
...
+```
+查看`$IOTDB_HOME/conf/jmx.password`，使用默认用户或者在此添加新用户
+若新增用户，编辑`$IOTDB_HOME/conf/jmx.access`，添加新增用户权限
 
-第一步：启动IoTDB server。
+第三步：启动IoTDB-server。
 
-第二步：建立连接。对于本地的监控，不需要手动配置连接。对于远程的监控，可以添加远程的ip地址，并将端口配置为`31999`。
+第四步：使用jvisualvm
+1. 确保安装jdk
```
Comment (qiaojialin) [inline docs/zh/UserGuide/6-System Tools/3-JMX Tool.md:49]:
```suggestion
1. 确保安装jdk 8，jdk 8以上需要下载 visualvm https://visualvm.github.io/download.html
```

Comment (samperson1997) [reply docs/zh/UserGuide/6-System Tools/3-JMX Tool.md]: Updated. Thanks : )

**Pull request merged**